tests/unit/CardUtilsTest.ts

```diff
@@ -379,6 +395,11 @@ describe('CardUtils', () => {
             const companyFeeds = getCompanyFeeds(undefined);
             expect(companyFeeds).toStrictEqual({});
         });
+
+        it('Should return only feeds that are not pending', () => {
+            const companyFeeds = getCompanyFeeds(cardFeedsCollection.FAKE_ID_4, false, true);
```

Comment:
@koko57 This should be 6 right? 
```suggestion
            const companyFeeds = getCompanyFeeds(cardFeedsCollection.FAKE_ID_6, false, true);
```

Reply:
@mountiny Could you please merge this PR because @koko57 will be OOO until the next week. Then I will make a quick PR to address this comment

Reply:
@VickyStash If possible, could you help to address above comment on this PR?

Reply:
@DylanDylann Yep, will do in a bit!

Reply:
@mountiny @DylanDylann Done!